vm-virtio/src/lib.rs

```diff
@@ -132,14 +203,40 @@ impl Translatable for u64 {
 }
 
 /// Helper for cloning a Queue since QueueState doesn't derive Clone
-pub fn clone_queue(queue: &Queue) -> Queue {
+pub fn clone_queue(
+    queue: &Queue,
+    #[cfg(all(feature = "mshv", feature = "sev_snp"))] access_platform: Option<
```

Comment:
Drop this - we already have an `Option` type surrounding it.

Reply:
Done.